docs/src/use/configure/ignore.md

```diff
@@ -8,135 +8,76 @@ eleventyNavigation:
 
 ---
```

Comment:
```suggestion
---

::: tip
This page explains how to ignore files using the flat config format. For the deprecated eslintrc format, [see the deprecated documentation](ignore-deprecated).
:::
```

Since other documents have this tip, I'm guessing this one should have it too.